### PR TITLE
Report: don't display score gauge header if there's only one.

### PR DESCRIPTION
**Pull request by paulirish**
Got tired of looking at tops of my perf/a11y reports that look like this:
image

This PR doesn't show the overview gauges if there's only one.

### CODE REVIEW

File: lighthouse-core/report/v2/renderer/report-renderer.js [hunk 1]
```diff
@@ -139,18 +139,21 @@ class ReportRenderer {
...
-    const scoreHeader = reportSection.appendChild(
-        this._dom.createElement('div', 'lh-scores-header'));
+    const scoreHeaderClass = `lh-scores-header ${isSoloCategory ? 'lh-scores-header__solo' : ''}`;
+    const scoreHeader = reportSection.appendChild(this._dom.createElement('div', scoreHeaderClass));
 
     const categories = reportSection.appendChild(this._dom.createElement('div', 'lh-categories'));
+
     for (const category of report.reportCategories) {
-      scoreHeader.appendChild(this._categoryRenderer.renderScoreGauge(category));
+      if (!isSoloCategory)
```
Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: I thought we didn't allow no braces multiline if statement

File: lighthouse-core/report/v2/renderer/report-renderer.js [hunk 2]
```diff
@@ -139,18 +139,21 @@ class ReportRenderer {
...
+    const isSoloCategory = report.reportCategories.length === 1;
 
     container.appendChild(this._renderReportHeader(report)); // sticky header goes at the top.
     container.appendChild(this._renderReportNav(report));
 
     const reportSection = container.appendChild(this._dom.createElement('div', 'lh-report'));
 
-    const scoreHeader = reportSection.appendChild(
-        this._dom.createElement('div', 'lh-scores-header'));
+    const scoreHeaderClass = `lh-scores-header ${isSoloCategory ? 'lh-scores-header__solo' : ''}`;
```
Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: Can we just not have a score header if < 2 categories?

Comment (paulirish) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: well i was thinking that too, but our styling uses that header's margin to get out from under the pos:fixed header.

File: lighthouse-core/report/v2/renderer/report-renderer.js (hunk 1 above)
Comment (paulirish) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: au contraire! 

(it was a surprise to me, too)

Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: #2263 :P

File: lighthouse-core/report/v2/renderer/report-renderer.js (hunk 2 above)
Comment (paulirish) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: ok so the better fix would be to put that margin on something else.
i guess that's the action necessary here.

**Comment by paulirish** [comment]
@brendankenny ptal

File: lighthouse-core/report/v2/renderer/report-renderer.js [hunk 3]
```diff
@@ -139,18 +139,22 @@ class ReportRenderer {
    */
   _renderReport(report) {
     const container = this._dom.createElement('div', 'lh-container');
+    const isSoloCategory = report.reportCategories.length === 1;
+    let scoreHeader;
```
Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/report-renderer.js]: move these two lines down to above `if (!isSoloCategory) {`?

**Comment by brendankenny** [comment]
style nit still applies but other than that, still looks good :P

**Pull request merged**